.github/workflows/publish_docs.yaml

```diff
@@ -1,6 +1,6 @@
-
 name: Deploy Docs
 on:
+  workflow_dispatch:
```

Comment:
why not leave it? it can be useful in the future and it shouldn't break anything?